tests/test_flows/test_flow_functions.py

```diff
@@ -283,9 +283,11 @@ def test_get_flow_reinstantiate_model_no_extension(self):
                                flow_id=10,
                                reinstantiate=True)
 
-    @unittest.skipIf(LooseVersion(sklearn.__version__) == "0.20.0",
-                     reason="No non-0.20 scikit-learn flow known.")
+    @unittest.skipIf(LooseVersion(sklearn.__version__) == "0.19.2",
+                     reason="Can't retrieve desired 0.20 run, "
+                            "see https://github.com/openml/OpenML/issues/992")
     def test_get_flow_reinstantiate_model_wrong_version(self):
-        # 20 is scikit-learn ==0.20.0
-        # I can't find a != 0.20 permanent flow on the test server.
-        self.assertRaises(ValueError, openml.flows.get_flow, flow_id=20, reinstantiate=True)
+        openml.config.server = self.production_server
+        _, sklearn_major, _ = LooseVersion(sklearn.__version__).version
+        flow = 8175 if sklearn_major != 19 else 8784
+        self.assertRaises(ValueError, openml.flows.get_flow, flow_id=flow, reinstantiate=True)
```

Comment:
I know it wasn't there, but could you please change this to an `assertRaisesRegex`?

Reply:
Done, also found that flow 8175  is `0.19.1` and not `0.19.2` so in terms of CI we can actually use it for all scikit-learn versions.